packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -16,6 +16,16 @@ import { getTeleports } from './util';
 //
 // IMPORTANT: Alphabetical based on text
 export const prodParasKusama: EndpointOption[] = [
+  {
+    info: 'aband',
+    homepage: 'https://a.band',
+    paraId: 2257,
+    text: 'Aband',
+    providers: {
+      'Aband DAO': 'wss://rpc-parachain.a.band',
+    }
+  },
+
```

Comment:
```suggestion
      'Aband DAO': 'wss://rpc-parachain.a.band'
    }
  },
```

Should fix the linting issues.